### PR TITLE
DOCS-719: Specify code identifiers in code blocks

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:
https://tigera.atlassian.net/browse/DOCS-719

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Review by bmckercher123 — APPROVED** [review]
LGTM

**Comment by girishshankaran** [comment]
@ctauchen Can you review this one?

File: calico-enterprise_versioned_docs/version-3.14/getting-started/upgrading/upgrading-calico-to-calico-enterprise/upgrade-to-tsee/standard.mdx [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: Steps to upgrade from open source Calico to Calico Enterprise.
 
 # Upgrade from Calico to Calico Enterprise
 
+```javascript
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.14/getting-started/upgrading/upgrading-calico-to-calico-enterprise/upgrade-to-tsee/standard.mdx]: STET

What happens here from line 7 to line 26 is just some JSX in our MDX file. In most places this kind of stuff happens in components. But here, for some reason, it was added directly. 

In any case, we want this code to stay as it was, here and everywhere else.

File: calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx [hunk 2]
```diff
@@ -122,7 +122,7 @@ As a last resort, create a new [Elasticsearch cluster](#how-to-create-a-new-clus
 
 **Solution/workaround**: Verify that the intrusion detection job is running, or try removing and reapplying:
 
-```
+```bash
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx]: Have you checked the output on blocks like this with the $ prompt? Do we want to remove?

Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx]: If we're happy to remove, a global s/$ kubectl/kubectl would probably do the trick.

**Review by ctauchen — COMMENTED** [review]
Very good, thanks! Only one problem, and one question.

File: calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx]: Yes, let's remove it. The $ symbol is redundant.

File: calico-enterprise_versioned_docs/version-3.14/getting-started/upgrading/upgrading-calico-to-calico-enterprise/upgrade-to-tsee/standard.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico-enterprise_versioned_docs/version-3.14/getting-started/upgrading/upgrading-calico-to-calico-enterprise/upgrade-to-tsee/standard.mdx]: Fixed.

File: calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx]: Fixed

File: calico-cloud/networking/egress/egress-gateway-maintenance.mdx
Comment (ctauchen) [inline calico-cloud/networking/egress/egress-gateway-maintenance.mdx:1]: There are quite a few instances where altering the $ kubectl line has resulted in strange output. This file, egress-gateway-maintenance.mdx, is its own kind of mess. I suggest you restore each instance of this file to its original and open a separate ticket to address the problems there.

But there are several place in other files where the $kubectl fix still leaves a second or third line with a prompt. For consistency, let's remove those in this round.

Comment (girishshankaran) [reply calico-cloud/networking/egress/egress-gateway-maintenance.mdx:1]: Sure, makes sense. Somehow, I missed spotting the instances with issues.

**Pull request closed without merging**